### PR TITLE
Remove global-zookeeper in documentation

### LINKED ISSUE #3305: "--global-zookeeper" already deprecated and hidden but still referenced in documentation
**Describe the bug**

`--global-zookeeper` has been deprecated and hidden since 2.0 release. However it is still
referenced in pulsar's documentation.

**Screenshots**

screen shot 2019-01-04 at 9 48 41 pm

**Additional context**

We should clean up the references of "--global-zookeeper" across pulsar's codebase and website.

### PR DESCRIPTION
**Pull request by liketic**
### Motivation

Replace all ```global zookeeper``` with  ```configuration store``` in all documents.

Relates to #3338
Closes #3305

### CODE REVIEW

**Comment by liketic** [comment]
retest this please

**Comment by sijie** [comment]
@liketic the pr looks good generally. however I would suggest splitting the changes into two separate PRs. the first PR focuses on documentation and website, making sure all the docs are using configuration store instead of global zookeeper; the second RP is removing the deprecated options or flags.

File: bin/pulsar [hunk 1]
```diff
@@ -297,11 +297,6 @@ elif [ $COMMAND == "bookie" ]; then
 elif [ $COMMAND == "zookeeper" ]; then
     PULSAR_LOG_FILE=${PULSAR_LOG_FILE:-"zookeeper.log"}
     exec $JAVA ${ZK_OPTS} $OPTS $ASPECTJ_AGENT -Dpulsar.log.file=$PULSAR_LOG_FILE org.apache.pulsar.zookeeper.ZooKeeperStarter $PULSAR_ZK_CONF $@
-elif [ $COMMAND == "global-zookeeper" ]; then
```
[the remark this answers is not in the document]
Comment (liketic) [reply bin/pulsar]: Thanks @merlimat , I opened #3338 for removing from codebase. And the command ```global-zookeeper``` won't be removed.

**Comment by sijie** [comment]
@liketic I should have told you first :) - you don't need to fix `site` directory, that was the old website. but that's okay

**Pull request merged**